src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs

```diff
@@ -740,26 +715,30 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
 
                     if (socketError == SocketError.Success)
                     {
-                        m_AcceptSocket = _currentSocket.UpdateAcceptSocket(m_AcceptSocket, _currentSocket._rightEndPoint.Create(remoteSocketAddress));
+                        _acceptSocket = _currentSocket.UpdateAcceptSocket(_acceptSocket, _currentSocket._rightEndPoint.Create(remoteSocketAddress));
 
-                        if (s_LoggingEnabled)
-                            Logging.PrintInfo(Logging.Sockets, m_AcceptSocket,
-          SR.Format(SR.net_log_socket_accepted, m_AcceptSocket.RemoteEndPoint, m_AcceptSocket.LocalEndPoint));
+                        if (s_loggingEnabled)
```

Comment:
Please add {} to all `if` blocks. (I know this is still acceptable but I find the practice dangerous).